### PR TITLE
Add Adpone adapter

### PR DESCRIPTION
**Pull request by seergiioo6**

### CODE REVIEW

File: exchange/adapter_map.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package exchange
 
 import (
 	"fmt"
+	"github.com/prebid/prebid-server/adapters/adpone"
```
Comment (guscarreon) [inline exchange/adapter_map.go]: Can we put this include in the next paragraph where all the adapter imports are?
```
  3 import (
  4     "fmt"
  5  -  "github.com/prebid/prebid-server/adapters/adpone"
  6     "net/http"
  7     "strings"
  8
  9     "github.com/prebid/prebid-server/adapters"
 10     ttx "github.com/prebid/prebid-server/adapters/33across"
 11     "github.com/prebid/prebid-server/adapters/adform"
 12     "github.com/prebid/prebid-server/adapters/adkernel"
 13     "github.com/prebid/prebid-server/adapters/adkernelAdn"
 14  +  "github.com/prebid/prebid-server/adapters/adpone"
 15     "github.com/prebid/prebid-server/adapters/adtelligent"
 16     "github.com/prebid/prebid-server/adapters/advangelists"
exchange/adapter_map.go
```

File: adapters/adpone/adpone.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+) {
+	openRTBRequestJSON, err := json.Marshal(openRTBRequest)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	requestToBidder := &adapters.RequestData{
```
[the remark this answers is not in the document]
Comment (seergiioo6) [reply adapters/adpone/adpone.go]: Absolutely yes; I will add more headers. Thanks a lot

File: adapters/adpone/adpone.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+func (adapter *adponeAdapter) MakeRequests(
+	openRTBRequest *openrtb.BidRequest,
+	reqInfo *adapters.ExtraRequestInfo,
+) (
+	requestsToBidder []*adapters.RequestData,
+	errs []error,
+) {
+	openRTBRequestJSON, err := json.Marshal(openRTBRequest)
```
[the remark this answers is not in the document]
Comment (seergiioo6) [reply adapters/adpone/adpone.go]: You're right... 

We need at least 1 Impression to return a valid response.

Thanks again.

File: static/bidder-info/adpone.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+maintainer:
+  email: "tech@adpone.com"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+  site:
+    mediaTypes:
+      - banner
```
Comment (guscarreon) [inline static/bidder-info/adpone.yaml]: Can you add a newline in here?

File: static/bidder-params/adpone.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Adpone Adapter Params",
+  "description": "A schema which validates params accepted by the adpone adapter",
+  "type": "object",
+  "properties": {},
```
Comment (guscarreon) [inline static/bidder-params/adpone.json]: Are we sure we don't have any parameters we want to check for? If not, do you think you'll need them in the future?

Comment (seergiioo6) [reply static/bidder-params/adpone.json]: Probably in the future we will need some parameters so I have added one (placementId). Thanks!

File: adapters/adpone/adpone.go [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+func (adapter *adponeAdapter) MakeRequests(
+	openRTBRequest *openrtb.BidRequest,
+	reqInfo *adapters.ExtraRequestInfo,
+) (
+	requestsToBidder []*adapters.RequestData,
+	errs []error,
+) {
+	var imp = &openRTBRequest.Imp[0]
```
Comment (guscarreon) [inline adapters/adpone/adpone.go]: When a request with an empty `openRTBRequest.Imp` array comes in, your adapter fails with an "index out of range" error in line 29. Can we check for `nil` before dereferencing the `Imp` array and check its length before accessing any of its elements?

**Pull request merged**